client/src/components/landing/landing.css

```diff
@@ -184,7 +184,8 @@ p.caption {
 }
 
 .map-ui .btn {
-  min-height: 80px;
+  height: 80px;
+  max-height: 80px;
```

Comment:
We should not set the height or max-height for the buttons. Since the text will wrap for smaller screens, setting the height would squeeze the text and reduce padding.

I suggest setting the min-height to the height of the largest button, that way they would all look the same on desktop. In smaller screens the text will wrap and the buttons will have different larger sizes which is ok.

Reply:
All buttons were less than 80px due to reformatting the height of the icons. So I set the min-height to 80px.